nixos/modules/services/monitoring/smartd.nix

```diff
@@ -108,6 +108,18 @@ in
         '';
       };
 
+      extraOptions = mkOption {
+        default = "";
+        type = types.str;
```

Comment:
How about ```types.listOf types.str```?

Reply:
I looked at several examples of an `extraOptions`... option, and there seems to be a roughly 50/50 split between a single string and lists of strings. Is there a strong reason for me to change this? Do core maintainers prefer one over the other?

Reply:
I like the list type it because it allows easily merging multiple definitions. But I cannot speak for others.

Reply:
I'm with @bjornfor on this for the same reason.